client/src/index.jsx

```diff
@@ -33,47 +33,49 @@ class Player extends React.Component {
 
   // }
 
-  initialize() {
+
+  componentDidMount() {
     $.ajax({
       type: "GET",
-      url: `http://52.37.102.63:3005/songdata/${this.songId}`,
+      url: `http://localhost:3005/songdata/${this.song_id}`,
       success: (res) => {
+        console.log('Response: ', res);
         this.setState({
           currentSong: this.audio = new Howl({
-            src: [res.songURL],
+            src: [res.data[0].song_url],
             // onplay: () => { requestAnimationFrame(this.frame) }
           }),
-          currentPicture: res.songImage,
-          currentSongName: res.songName
+          currentPicture: res.data[0].song_image,
+          currentSongName: res.data[0].song_name
+        }, () => {
+          console.log('Updated state: ', this.state);
         })
+      },
+      error: (err) => {
+        console.log(err);
       }
     })
```

Comment:
If this is data your service owns, consider whether this call would be better in your `constructor` so that the request can be made as early as possible.

Reply:
React docs actually tell you not to setState inside of the constructor: https://reactjs.org/docs/react-component.html#constructor